charts/opensearch-cluster/README.md

```diff
@@ -0,0 +1,109 @@
+| cluster.dashboards.resources | object | `{}` | dashboards pod cpu and memory resources |
+| cluster.dashboards.securityContext | object | `{}` | dashboards security context configuration |
+| cluster.dashboards.service.loadBalancerSourceRanges | list | `[]` | source ranges for a loadbalancer |
+| cluster.dashboards.service.type | string | `"ClusterIP"` | dashboards service type |
+| cluster.dashboards.tls.caSecret | object | `{}` | Secret that contains the ca certificate as ca.crt. If this and generate=true is set the existing CA cert from that secret is used to generate the node certs. In this case must contain ca.crt and ca.key fields |
+| cluster.dashboards.tls.enable | bool | `false` | Enable HTTPS for dashboards |
+| cluster.dashboards.tls.generate | bool | `true` | generate certificate, if false secret must be provided |
+| cluster.dashboards.tls.secret | string | `nil` | Optional, name of a TLS secret that contains ca.crt, tls.key and tls.crt data. If ca.crt is in a different secret provide it via the caSecret field |
+| cluster.dashboards.tolerations | list | `[]` | dashboards pod tolerations |
+| cluster.dashboards.version | string | `"2.3.0"` | dashboards version |
+| cluster.general.additionalConfig | object | `{}` | Extra items to add to the opensearch.yml |
+| cluster.general.additionalVolumes | list | `[]` | Additional volumes to mount to all pods in the cluster. Supported volume types configMap, emptyDir, secret (with default Kubernetes configuration schema) |
+| cluster.general.drainDataNodes | bool | `true` | Controls whether to drain data notes on rolling restart operations |
+| cluster.general.httpPort | int | `9200` | Opensearch service http port |
+| cluster.general.image | string | `"docker.io/opensearchproject/opensearch"` | Opensearch image |
+| cluster.general.imagePullPolicy | string | `"IfNotPresent"` | Default image pull policy |
+| cluster.general.keystore | list | `[]` | Populate opensearch keystore before startup |
+| cluster.general.monitoring.enable | bool | `false` | Enable cluster monitoring |
+| cluster.general.monitoring.monitoringUserSecret | string | `""` | Secret with 'username' and 'password' keys for monitoring user. You could also use OpenSearchUser CRD instead of setting it. |
+| cluster.general.monitoring.pluginUrl | string | `""` | Custom URL for the monitoring plugin |
+| cluster.general.monitoring.scrapeInterval | string | `"30s"` | How often to scrape metrics |
+| cluster.general.monitoring.tlsConfig | object | `{}` | Override the tlsConfig of the generated ServiceMonitor |
+| cluster.general.pluginsList | list | `[]` | List of Opensearch plugins to install |
+| cluster.general.podSecurityContext | object | `{}` | Opensearch pod security context configuration |
+| cluster.general.securityContext | object | `{}` | Opensearch securityContext |
+| cluster.general.serviceAccount | string | `""` | Opensearch serviceAccount name |
```

Comment:
It should be documented explicitly that if a serviceaccount is set/created via `serviceAccount.name` it must be added here to be used by the cluster.

Reply:
Thanks @swoehrl-mw, @evheniyt please take care of and I will merge the PR after the CI's pass.